pkg/tsdb/cloudmonitoring/time_series_query.go

```diff
@@ -180,6 +171,24 @@ func (timeSeriesQuery cloudMonitoringTimeSeriesQuery) parseResponse(queryRes *pl
 				continue
 			}
 
+			labelsByKey := make(map[string][]string)
+			for key, values := range labels {
+				for value := range values {
+					labelsByKey[key] = append(labelsByKey[key], value)
+				}
+			}
+
+			if frame.Meta != nil {
```

Comment:
Can we initialize meta only if nil? Then the else statement can be removed.

Reply:
What if meta is set previously already with some other info?